Resources/Prototypes/Catalog/Fills/Crates/fun.yml

```diff
@@ -7,27 +7,49 @@
   - type: StorageFill
     contents:
       - id: PlushieBee
+        prob: 0.1
       - id: PlushieNar
+        prob: 0.1
       - id: PlushieCarp
+        prob: 0.1
       - id: PlushieNuke
+        prob: 0.1
       - id: PlushieSlime
+        prob: 0.1
       - id: PlushieSnake
+        prob: 0.1
       - id: PlushieLizard
+        prob: 0.1
       - id: PlushieSpaceLizard
+        prob: 0.1
       - id: PlushieVox
+        prob: 0.1
       - id: PlushieRouny
+        prob: 0.1
       - id: PlushieRatvar
+        prob: 0.1
       - id: PlushieSharkBlue
+        prob: 0.1
         orGroup: PlushieShark
       - id: PlushieSharkGrey
+        prob: 0.1
         orGroup: PlushieShark
       - id: PlushieAtmosian
+        prob: 0.1
       - id: PlushieDiona
+        prob: 0.1
       - id: PlushieXeno
+        prob: 0.1
       - id: PlushieHampter
+        prob: 0.1
       - id: PlushieMoth
+        prob: 0.1
       - id: PlushieArachind
+        prob: 0.1
       - id: PlushiePenguin
+        prob: 0.1
+      - id: PlushieScientist
+        prob: 0.1
```

Comment:
This gives the crate a chance of spawning absolutely nothing which really shouldn't be happening.

Reply:
Plush casino) Alright I will return it as it was before